com.basiomeuspuga.Lector.json

```diff
@@ -0,0 +1,143 @@
+{
+	"app-id":"com.basiomeuspuga.Lector",
```

Comment:
this domain name doesn't exist? com.github.BasioMeusPuga.Lector might be better (or io.github.BasioMeusPuga.Lector if you have a website there) a bit more on app-id selection http://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
Understood. This is my first time building one of these. Thank you for being patient.